protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,81 +110,79 @@ Select {
 
-### External Oracles - Deletions
+### External Oracles - Deactivation
 
-1. Aligned with the existing logic, when no market listens to a data source, whatever that source is, it is automatically disregarded / deleted from the engine. Same applies for ethereum oracles (<a name="0082-ETHD-008" href="#0082-ETHD-008">0082-ETHD-008</a>)
-2. When ethereum oracle is referenced / used by an existing market, it should not be deleted (<a name="0082-ETHD-009" href="#0082-ETHD-009">0082-ETHD-009</a>)
-3. If a single data source is used by multiple markets, then should NOT be able to delete the data source even if one of those markets is actively using the data source (<a name="0082-ETHD-010" href="#0082-ETHD-010">0082-ETHD-010</a>)
+1. Aligned with the existing logic, when no market listens to a data source, whatever that source is, it is automatically disregarded by the engine and the status of the data source is set to DEACTIVATED. Same applies for ethereum oracles (<a name="0082-ETHD-007" href="#0082-ETHD-007">0082-ETHD-007</a>)
+2. When ethereum oracle is referenced / used by an existing market, it should not be possible to DEACTIVATE that ethereum oracle data source without either a termination or settlement transaction being sent to the core and successfully processed.(<a name="0082-ETHD-008" href="#0082-ETHD-008">0082-ETHD-008</a>)
+3. If a single data source is used by multiple markets, then it should NOT be possible to DEACTIVATE the data source even if one of those markets is actively using the data source (<a name="0082-ETHD-009" href="#0082-ETHD-009">0082-ETHD-009</a>)
 
 ### External Oracles - Validations
 
-1. Validate if the smart contract address is valid (<a name="0082-ETHD-011" href="#0082-ETHD-011">0082-ETHD-011</a>)
-2. Validate if the data elements of the oracle data source is valid - e.g. source for a value that's returned as boolean but have a filter / condition for greater than 0 (<a name="0082-ETHD-012" href="#0082-ETHD-012">0082-ETHD-012</a>)
-3. Validations for min / max frequency of listening for events / read a smart contract (<a name="0082-ETHD-013" href="#0082-ETHD-013">0082-ETHD-013</a>)
-4. When a proposal that uses ethereum oracles, defines incorrect data (contract address, ABI) the system should throw an error and the proposal should not pass validation (<a name="0082-ETHD-014" href="#0082-ETHD-014">0082-ETHD-014</a>)
-5. Any mismatch between expected fields and received fields should emit an error via the TX RESULT event (<a name="0082-ETHD-016" href="#0082-ETHD-016">0082-ETHD-016</a>)
+1. Validate if the smart contract address is valid (<a name="0082-ETHD-010" href="#0082-ETHD-010">0082-ETHD-010</a>)
+2. Validate if the data elements of the oracle data source is valid - e.g. source for a value that's returned as boolean but have a filter / condition for greater than 0 (<a name="0082-ETHD-011" href="#0082-ETHD-011">0082-ETHD-011</a>)
```

Comment:
Can we clarify this further: what we actually can validate so its a real AC with something clear to assert in the test?

suggestted example
Call the contract contract and see if the types in the ABI match whats in the oracle spec

Reply:
AC amended as per comment.